### DISCUSSION TITLE
Automation

### DISCUSSION THREAD

**Discussion by mattiamarras** [chatwoot/chatwoot#8128] [General]
Is it possible to set an automation rules for audio whatsapp message?
I would like to respond that the service is not avalaible and that we respond only to text messages.

**Comment by hermesalvesbr** [comment]
Have you managed to advance this automation? I need this too.

**Comment by mattiamarras** [comment]
> Have you managed to advance this automation? I need this too.
No

**Reply by pranavrajs** [reply]
@mattiamarras Missed this thread. Can you please provide more context here? Are you looking to setup an automation for audio only message or are you looking to provide a responder for a call?

**Reply by mattiamarras** [reply]
Ciao Pranav. My customer care does not hear audio messages and responds to those messages with a canned response like "this is a chat service, we cannot hear your message, please write your question in the chat". I would like to automate this operation.
